Casks/font/font-s/font-sf-mono-nerd-font-ligaturized.rb

```diff
@@ -0,0 +1,23 @@
+cask "font-sf-mono-nerd-font-ligaturized" do
+  version :latest
+  sha256 :no_check
+
+  url "https://github.com/shaunsingh/SFMono-Nerd-Font-Ligaturized/archive/refs/heads/main.tar.gz"
+  name "SF Mono Nerd-Font-Ligaturized"
```

Comment:
```suggestion
  name "SF Mono Nerd Font Ligaturized"
```

I would keep it consistent with the Nerd Font casks and remove the hyphens.

Reply:
FIne with me but this is not from Nerd Font official site, don't know if that matters tho.

Reply:
The Nerd Fonts project is generally written without a hyphen, so I would do the same here. Right now, the hyphens seem somewhat arbitrary. I don't feel too strongly about it, though.